models/claims_preprocessing/encounters/final/acute_inpatient__encounter_grain.sql

```diff
@@ -0,0 +1,198 @@
+{{ config(
+     enabled = var('claims_preprocessing_enabled',var('claims_enabled',var('tuva_marts_enabled',False))) | as_bool
+   )
+}}
+
+with detail_values as (
+    select stg.*
+    ,cli.encounter_id
+    ,cli.old_encounter_id
+    ,ed.encounter_start_date
+    ,ed.encounter_end_date
+      ,cli.encounter_type
+    ,cli.encounter_group
+    from  {{ ref('encounters__stg_medical_claim') }} stg
+    inner join {{ ref('encounters__combined_claim_line_crosswalk') }} cli on stg.claim_id = cli.claim_id  --replace this ref with the deduped version when complete
+    and
+    stg.claim_line_number = cli.claim_line_number
+    and
+    cli.encounter_type = 'acute inpatient'
+    and
+    cli.claim_line_attribution_number = 1
```

Comment:
I think this is OK actually but might need to check with Brad to confirm. I'm not seeing duplicates in the underlying table at least on CMS synthetic data.